logging/zap/grpclogger.go

```diff
@@ -47,14 +47,16 @@ func (l *zapGrpcLogger) Println(args ...interface{}) {
 	l.logger.Info(fmt.Sprint(args...))
 }
 
-// ReplaceGrpcLoggerV2 replaces the grpc_log.LoggerV2 with the provided logger.
-// It should be called before any gRPC functions.
+// ReplaceGrpcLoggerV2 replaces the grpclog.LoggerV2 with the provided logger.
+// It should be called before any gRPC functions. Logging verbosity defaults to info level.
+// To adjust gRPC logging verbosity, see ReplaceGrpcLoggerV2WithVerbosity.
 func ReplaceGrpcLoggerV2(logger *zap.Logger) {
 	ReplaceGrpcLoggerV2WithVerbosity(logger, 0)
 }
 
-// ReplaceGrpcLoggerV2WithVerbosity replaces the grpc_.LoggerV2 with the provided logger and verbosity.
+// ReplaceGrpcLoggerV2WithVerbosity replaces the grpclog.Logger with the provided logger and verbosity.
 // It should be called before any gRPC functions.
+// verbosity correlates to grpclogs verbosity levels. A higher verbosity value results in less logging.
```

Comment:
Would it be too fragile to put a link to the verbosity levels in grpclog? They aren't exported.